crypto/src/address.rs

```diff
@@ -6,6 +6,8 @@ use bech32::{FromBase32, ToBase32, Variant};
 
 use crate::{fmd, ka, keys::Diversifier, Fq};
 
+pub const CURRENT_CHAIN_ID: &str = "penumbra_tn001";
```

Comment:
nit: from what i've seen, most of the other chain IDs have `-` separators, maybe worth aligning on that?